### DISCUSSION TITLE
Removing metronome reset sound in timing panel

### DISCUSSION THREAD

**Discussion by Reletr** [ppy/osu#30253] [Ideas]
In the timing panel of the editor, when you pause the playback of the song, the metronome ticker will reset itself to its center position, making a sharp ticking sound.

https://github.com/user-attachments/assets/06f9a3bc-b0d8-414e-9e10-a645ec6095d0

It feels unneccesary to have this, since the sound doesn't signify anything useful for me while editing, and it can get pretty annoying and distracting while timing variable bpm songs, where pausing song playback to make adjustments happens frequently.

**Comment by peppy** [comment]
This is intentional. It only plays when nothing else is playing so I can't see how it can be that distracting.
